docs/streams.html

```diff
@@ -279,8 +279,8 @@
 <pre>
     KStreamBuilder builder = new KStreamBuilder();
 
-    KStream<String, GenericRecord> source1 = builder.stream("topic1", "topic2");
-    KTable<String, GenericRecord> source2 = builder.table("topic3", "stateStoreName");
+    KStream&lt;String, GenericRecord&gt; source1 = builder.stream("topic1", "topic2");
```

Comment:
That didn't work when I tried it. With only `<code>` the code wraps (it ignores the EOL character), and it still doesn't show the generics. With both `<pre>` and `<code>` it still doesn't show the generics.